### DISCUSSION TITLE
Build output cannopt be found during YAML script run

### DISCUSSION THREAD

**Discussion by teacherjoseluis** [codemagic-ci-cd/codemagic-docs#2233] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Android (Kotlin)

### Steps to reproduce

I have tried different configurations in my YAML script to wait for the aab file to be created so it can be signed and published to Google Play store but the script always fails as it seems like it's trying to look for it but never find it.

The strange thing is that I have tried as well logging into the build machine and I was able to confirm after the YAML script fails, that indeed the aab file is created at the CM_BUILD_OUTPUT_DIR location

### Expected results

I'm expecting the script to use the output aab file and continue with the signing process

### Actual results

It looks like the script is unable to find it, even when adding a while loop that keeps on waiting for the file to be created

### Build id (optional)

6569282b5c74e3c12a9eefdb

**Comment by himesh-cm** [comment, marked as answer]
Hi @teacherjoseluis, the artifact at `CM_BUILD_OUTPUT_DIR ` is available; once the publishing step is completed, the above build ID doesn't reach that stage. 
You can refer to `- app/build/outputs/**/*.aab` for your use case.
